### PR TITLE
Tieshia

### PR DESCRIPTION
**Pull request by Tieshia**
### WHAT are you trying to accomplish with this PR?
Finished user edit, update, index, and destroy actions.

### Checklist

- [X] It is safe to simply rollback this change. OR It is safe to rollback this change until PR XX (maybe in another repo) has been merged.
- [X] I have reviewed the failure scenarios of this approach. E.g. What happens if an API call fails?
- [X] I have :tophat:'d these changes following this project's guidelines.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class User < ApplicationRecord
   validates :email, presence: true, length: { maximum: 255 },
                     format: { with: VALID_EMAIL_REGEX },
                     uniqueness: { case_sensitive: false }
-  validates :password, presence: true, length: { minimum: 6 }
+  validates :password, presence: true, length: { minimum: 6 }, allow_nil: true
```
Comment (ChanChar) [inline app/models/user.rb:10]: intentional?

**Review by ChanChar — APPROVED** [review]
It's all coming together! 

Let's try to name our PRs as the context of the changes.

File: app/models/user.rb (hunk 1 above)
Comment (Tieshia) [reply app/models/user.rb:10]: Yes, apparently it allows for users to update their information without having to change their password every time. Can we test it out tomorrow to see if it's absolutely necessary?

Comment (ChanChar) [reply app/models/user.rb:10]: interesting, this validation makes it seem like we're allowing for nil passwords.

**Pull request merged**